lib/utils/kube.go

```diff
@@ -50,21 +49,6 @@ func GetPodByName(clientset *kubernetes.Clientset, podName string) (*v1.Pod, err
 	return podsInterface.Get(context.Background(), podName, metav1.GetOptions{})
 }
 
-func GetPods(clientset *kubernetes.Clientset, lbls map[string]string) ([]v1.Pod, error) {
```

Comment:
why remove this function and shift it to deploy.go? This file contains all kube related util functions.